package.json

```diff
@@ -1,67 +1,70 @@
 {
-  "name": "@salesforce/plugin-template-sf",
-  "description": "A template repository for sf plugins",
-  "version": "0.0.1",
+  "name": "@salesforce/plugin-settings",
+  "description": "configure the Salesforce CLI",
+  "version": "1.0.0",
   "author": "Salesforce",
   "bugs": "https://github.com/forcedotcom/cli/issues",
   "dependencies": {
-    "@oclif/core": "^1.9.5",
-    "@salesforce/core": "^3.25.0",
-    "@salesforce/kit": "^1.5.45",
-    "@salesforce/sf-plugins-core": "^1.13.0",
+    "@oclif/core": "^1.13.10",
+    "@salesforce/core": "^3.26.1",
+    "@salesforce/sf-plugins-core": "^1.13.2",
     "tslib": "^2"
   },
   "devDependencies": {
     "@oclif/plugin-command-snapshot": "^3.1.3",
-    "@oclif/test": "^2.1.0",
-    "@salesforce/cli-plugins-testkit": "^1.5.35",
+    "@oclif/test": "^2.1.1",
+    "@salesforce/cli-plugins-testkit": "^2.3.10",
     "@salesforce/dev-config": "^3.1.0",
-    "@salesforce/dev-scripts": "^2",
+    "@salesforce/dev-scripts": "^2.0.4",
```

Comment:
there's a newer dev-scripts that updates the general (non-plugin) linter rules.  Using that now saves some extra work later.

It'll bump some other deps and clean up unused deps, too.